internal/manifests/var.go

```diff
@@ -131,10 +141,38 @@ func serviceNameQueryFrontendHTTP(stackName string) string {
 	return fmt.Sprintf("loki-query-frontend-http-%s", stackName)
 }
 
+func signingServiceSecretName(stackName string) string {
+	return fmt.Sprintf("%s-%s", stackName, "metrics")
+}
```

Comment:
:x:  The implementation here makes a wrong assumption namely that secret's name is based on `stackName` and a suffix `-metrics`. However the cert signing service will create one secret per Loki component service, e.g.
- Distributor service: `loki-distributor-http-my-stack-name` --> Secret: `loki-distributor-http-my-stack-name-metrics`
- Ingester service: `loki-ingester-http-my-stack-name` --> Secret: `loki-ingester-http-my-stack-name-metrics`

The approarch to take here is to pass the service name per component here.